### PR TITLE
✨ Add Database Connection With Stub Data

### PR DESCRIPTION
**Pull request by connormaglynn**
## 👀 Purpose

- In relation to https://github.com/ministryofjustice/operations-engineering/issues/4620
- To add database integration for the application

## ♻️ What's changed

- Added code to connect and populate the database with some stub data
- Refactored application stub data with stub data that lives in the live database
- Added a base Cronjob template, which will be enhanced to store the mapping of assets to owners on a schedule

## 📝 Notes

- NA

### CODE REVIEW

File: .github/workflows/cicd-deploy-to-dev.yml [hunk 1]
```diff
@@ -14,8 +14,13 @@ env:
   IMAGE_TAG: ${{ github.sha }}
   ECR_REGISTRY: ${{ vars.DEVElOPMENT_ECR_REGISTRY }}
   ECR_REPOSITORY: ${{ vars.DEVElOPMENT_ECR_REPOSITORY }}
+  ADMIN_GITHUB_TOKEN: ${{ secrets.DEV_ADMIN_GITHUB_TOKEN }}
```
Comment (tamsinforbes) [inline .github/workflows/cicd-deploy-to-dev.yml:17]: Where is this token generated? Shouldn't this be a bot account PAT following the naming convention for these?

File: Pipfile [hunk 2]
```diff
@@ -7,13 +7,11 @@ name = "pypi"
...
 flask = "==3.0.2"
 flask-cors = "==4.0.1"
 flask-limiter = "==3.5.0"
 govuk-frontend-jinja = "==3.0.0"
 gunicorn = "==22.0.0"
-sentry-sdk = "==1.39.2"
 pandas = "==2.2.2"
 plotly = "==5.23.0"
-
+psycopg2-binary = "2.9.9"
```
Comment (tamsinforbes) [inline Pipfile:17]:
```suggestion
psycopg2-binary = "==2.9.9"
```
???

**Review by tamsinforbes — APPROVED** [review]
Couple of comments

File: .github/workflows/cicd-deploy-to-dev.yml (hunk 1 above)
Comment (connormaglynn) [reply .github/workflows/cicd-deploy-to-dev.yml]: Good spot, the token has not been generated yet so removed the reference for now 😁

File: .github/workflows/cicd-deploy-to-dev.yml [hunk 3]
```diff
@@ -65,6 +69,11 @@ jobs:
             --namespace ${KUBE_NAMESPACE} \
             --values=helm/find-a-github-repository-owner/values-dev.yaml \
             --set app.deployment.env.APP_SECRET_KEY=${FLASK_APP_SECRET} \
-            --set app.deployment.env.SENTRY_DSN_KEY=${SENTRY_DSN_KEY} \
+            --set app.deployment.env.ADMIN_GITHUB_TOKEN=${ADMIN_GITHUB_TOKEN} \
```
Comment (tamsinforbes) [inline .github/workflows/cicd-deploy-to-dev.yml:72]:
```suggestion
          
```

**Review by tamsinforbes — CHANGES_REQUESTED** [review]
Spotted another ADMIN_GITHUB_TOKEN lurking I think

**Review by tamsinforbes — APPROVED** [review]
#
🌝🌖🚀LGTM🚀🌘🌚

**Pull request merged**